frontend/src/views/consumer/event-listing/TicketInputSet.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { FlexBoxHorizontal } from './EventBooking';
+import { observer } from 'mobx-react';
+import { clamp } from '../../../utils';
+import Form, { InputField } from '../../../components/form';
+
+export default observer(
+  class TicketInputSet extends Component {
+    findTicketCatalogMaxSeats = id => {
+      const catalog = this.props.ticketCatalog.find(elem => elem.id == id);
+      if (catalog == undefined) {
```

Comment:
use `===` instead of `==`

Reply:
=== will break this code

Reply:
id is a number. it needs value based evaluation instead of identity